Движение ЦМ горизонтальный полёт и набор высоты

Erziaikina opened the pull request:
Тестила пока только горизонтальный полёт (но самолёт у меня почему-то падает)

Erziaikina committed:
Create practika.slx

Обычная модель математического маятника и маятника с затуханием

Erziaikina committed:
Create practika2.slx

Erziaikina committed:
Сохранение в предыдущей версии

Erziaikina committed:
Исправление записи SDI, добавила GoTo

Erziaikina committed:
Добавила вынужденные колебания

Erziaikina committed:
.

Erziaikina committed:
Create Model_plane.slx

Модель самолёта горизонтальный полёт

Erziaikina committed:
Update Model_plane.slx

Erziaikina committed:
.

Erziaikina committed:
Нашла ошибки, теперь должно работать правильно

Erziaikina committed:
Исправила некоторые ошибки, но самолёт продолжает падать, развивая при этом сверхзвуковую скорость. Не понимаю в чё не так

Erziaikina committed:
Create Model_plane5Old_vers.slx

Erziaikina committed:
Update Model_plane5Old_vers.slx

Erziaikina committed:
.

Erziaikina committed:
Update Model_plane5Old_vers.slx

Erziaikina committed:
Update Model_plane5Old_vers.slx

Erziaikina commented:
Исправила все ошибки, теперь может как держать высоту, так и её снижать или набирать

Erziaikina committed:
Update Model_plane5Old_vers.slx

Erziaikina committed:
.

Erziaikina committed:
Внесла правки в названия переменных и перенесла их в Model workspace

Erziaikina committed:
Добавила управление углом тангажа через угол наклона руля высоты.

Есть ошибка, тк угол тангажа не приходит в заданное значение.

Erziaikina committed:
Контур управления углом тангажа через изменение угла наклона руля высоты

Erziaikina committed:
.

NoMonsters reviewed (changes requested):
Привет.

Можешь пожалуйста:
1. Создать новую ветку согласно правилам нейминга см. https://github.com/NoMonsters/studentsFCS/issues/5 . 
2. Перенести туда все коммиты из текущей ветки (см. merge и rebase). 
3. Закрыть этот пулл реквест
4. Удалить ветку с неправильным названием и создать заново пулреквест из новой ветки.

Erziaikina committed:
Переделала модель и контуры управления по скорости и высоте

В модели добавила возможность отклонения на угол рыскания и угол крена (если узнать коэффициенты для моментов Mx и My). Переделала контуры управления по скорости и высоте. Коэффициенты не идеальные, но к нужным значениям приходит и самолёт не падает.
